charts/argo-rollouts/templates/controller/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         {{- if not .Values.clusterInstall }}
         - --namespaced
         {{- end }}
-        {{- if gt .Values.controller.replicas 1.0 }}
+        {{- if gt .Values.controller.replicas 1 }}
```

Comment:
On my local, I got error. 😢 

```
$ helm template . > foo.yaml

Error: template: argo-rollouts/templates/controller/deployment.yaml:51:15: executing "argo-rollouts/templates/controller/deployment.yaml" at <gt .Values.controller.replicas 1>: error calling gt: incompatible types for comparison
```

Reply:
Fixed

Reply:
What's nice is our [CI caught it[(https://github.com/argoproj/argo-helm/actions/runs/6625851696/job/17997619786#step:7:46). 😄 

Ran that locally and was able to reproduce. CI ftw.